src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -1076,9 +1077,9 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             Guid deterministicGuid;
             if (!Guid.TryParse(obj.Value<string>(), out deterministicGuid))
             {
-                Console.WriteLine("the id was not a guid, converting to a guid");
+                Debug.WriteLine("the id was not a guid, converting to a guid");
```

Comment:
@mjkkirschner Can you remind me the consequence of this change, if I remember correctly this will reduce the console log size we see in Jenkins correct?

Reply:
yes, we should not get logs filled with random guids - these were useful when we were creating this functionality, but not so much anymore.